stdlib/seq.mc

```diff
@@ -31,6 +43,13 @@ let all = fix (lam all. lam p. lam xs.
   then true
   else and (p (nth xs 0)) (all p (slice xs 1 (length xs))))
 
+-- Append and concat
+
+let append = lam seq1. lam seq2.
+    let f = lam acc. lam x. cons x acc in
+    foldr f seq2 seq1
+
+let concat = lam seqs. foldl append [] seqs
```

Comment:
`append` already exists in the built-ins with the name `concat`. I suggest replacing this chunk by:
```
let join = lam  seqs. foldl concat [] seqs
```